lua/plugins/cmp.lua

```diff
@@ -60,6 +60,12 @@ return {
         mapping = {
           ["<Up>"] = cmp.mapping.select_prev_item { behavior = cmp.SelectBehavior.Select },
           ["<Down>"] = cmp.mapping.select_next_item { behavior = cmp.SelectBehavior.Select },
+          ["<PageUp>"] = cmp.mapping.select_prev_item { behavior = cmp.SelectBehavior.Select, count=8 },
+          ["<PageDown>"] = cmp.mapping.select_next_item { behavior = cmp.SelectBehavior.Select, count=8 },
+          ["<C-PageUp>"] = cmp.mapping.select_prev_item { behavior = cmp.SelectBehavior.Select, count=16 },
+          ["<C-PageDown>"] = cmp.mapping.select_next_item { behavior = cmp.SelectBehavior.Select, count=16 },
```

Comment:
```suggestion
```

We should remove `<C-PageUp>` and `<C-PageDown>` as this is canonically used for navigating tabs and this is not what people would expect.